backend/app/routes/resources/addResource.js

```diff
@@ -0,0 +1,46 @@
+//Getting resource modal
+const Resource = require('../../models/resource');
+//Getting resource handler
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  try {
+    //Destructuring fields which are must
+    const { name, email, url, trustLevel, expiryDate } = req.body;
```

Comment:
We use the validation Schema to add check for the required field. Please see https://github.com/HITK-TECH-Community/Community-Website/pull/695/files#diff-66eaadabdfb2a1e580497efec3551b7ff1a144afc8f0d2732ba5643b0032f3c3R1 this pr for reference